cd/auto-generate/auto-generate-controllers.sh

```diff
@@ -228,7 +229,8 @@ for CONTROLLER_NAME in $CONTROLLER_NAMES; do
 
     # Add all the files & create a GitHub commit
     git add .
-    COMMIT_MSG="Update ACK runtime to \`$ACK_RUNTIME_VERSION\`"
+    COMMIT_MSG="Update ACK runtime to \`$ACK_RUNTIME_VERSION\`
+    release notes: $CODEGEN_RELEASES/tag/$ACK_RUNTIME_VERSION"
```

Comment:
Please add an empty line after the first line to break up the commit summary with the details. Also would you mind capitalising "Release". Thanks!